### ISSUE TITLE
0.8.3 regression

### ISSUE THREAD

**Issue by mdantonio** [sbdchd/celery-types#39]
Hello,
just after the release of version 0.8.3 my nightly mypy CI build failed due to errors in celery typing

Once pinned celery-types to 0.8.2 everything back working

May it be some regression introduced in 0.8.3? It is seems that with the new version celery is no longer recognized as typed

```
error: Skipping analyzing "celery.result": found module but no type hints or library stubs
error: Skipping analyzing "celery.app.task": found module but no type hints or library stubs
error: Argument 1 to "xyz" becomes "Any" due to an unfollowed import
error: Skipping analyzing "celery": found module but no type hints or library stubs
error: Skipping analyzing "celery.exceptions": found module but no type hints or library stubs
```

**Comment by sbdchd** [comment]
Should be fixed in version 0.9.0 of celery-types

**Comment by mdantonio** [comment]
Confirmed, thank you
